ql/src/test/results/clientpositive/llap/analyze_npe.q.out

```diff
@@ -159,7 +159,7 @@ STAGE PLANS:
           Filter Operator
             predicate: c1 is null (type: boolean)
             Select Operator
-              expressions: null (type: void)
+              expressions: array(null) (type: array<string>)
```

Comment:
I understand the changes in this file but in some databases such nuances (`null` vs `array(null)`) lead to different results.
I run the following on Postgres and confirmed that plain null and an array with nulls is not the same thing.
```
create table tbl_arr(ca INT ARRAY);
insert into tbl_arr values ('{1}');
insert into tbl_arr values ('{2,3}');
insert into tbl_arr values (null);
insert into tbl_arr values ('{null}');

select * from tbl_arr;
select * from tbl_arr where ca is null;
select * from tbl_arr where ca[1] is null;
```
Can you please check what Hive does/should do and if the results are different after your changes. This file has just `EXPLAIN` statements so not easy to tell.